modules/local/matricescreation.nf

```diff
@@ -40,6 +40,7 @@ process MATRICESCREATION {
     design_matrix[treatment_samples, name] <- 1
     design_matrix[treatment_samples, paste0(gsub(',', '_', '$meta.treatment'),"_vs_",gsub(",","_",'$meta.reference'))] <- 1
 
+    colnames(design_matrix) <- c("Samples", "baseline", name)
```

Comment:
I don't follow why we have to define the column names again if they are set when the matrix is created

Reply:
Hi, sorry  @adamrtalbot for the lack of explanation!
The bug I am currently having is due to dashes in potential sample names. Anything with a "-" in the sample name is converted to a dot in matrix and table creation in R and I haven't found a workaround except explicitely renaming the columns afterwards.
I could also rename the column names temporarily when i create the design_matrix and then rename with a comment, i just don't find anything very elegant sofar..

`    design_matrix <- data.frame(matrix(0, nrow = length(all_samples), ncol = 3,
                                dimnames = list(all_samples,
                                                c("temp1", "temp2",
                                                    temp3))))`